chromium_src/net/tools/transport_security_state_generator/input_file_parsers.cc

```diff
@@ -34,6 +34,7 @@ constexpr base::StringPiece kBravePinsJson = R"brave_pins_json({
   ],
   "entries": [
     // Brave
+    { "name": "ai-chat.proxy.brave.com", "pins": "brave"},
```

Comment:
nit: moving it one line down would preserve the alphabetical order

Reply:
done